File: TransCoder_tokenized_test_set_functions/java/MAXIMUM_CONSECUTIVE_REPEATING_CHARACTER_STRING_1.java
static char maxRepeating ( String str ) {
  int n = str . length ( );
  int count = 0;
  char res = str . charAt ( 0 );
  int cur_count = 1;
  for ( int i = 0;
  i < n;
  i ++ ) {
    if ( i < n - 1 && str . charAt ( i ) == str . charAt ( i + 1 ) ) cur_count ++;
    else {
      if ( cur_count > count ) {
        count = cur_count;
        res = str . charAt ( i );
      }
      cur_count = 1;
    }
  }
  return res;
}

File: results/py/MAXIMUM_CONSECUTIVE_REPEATING_CHARACTER_STRING_1.cpp
string f_filled ( string str ) {
  int n = str . length ( ) ;
  int count = 0 ;
  string res = str [ 0 ] ;
  int curCount = 1 ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    if ( ( i < n - 1 && str [ i ] == str [ i + 1 ] ) || ( i == n - 1 && str [ i ] == str [ i + 1 ] ) ) {
      curCount ++ ;
    }
    else {
      if ( curCount > count ) {
        count = curCount ;
        res = str [ i ] ;
      }
      curCount = 1 ;
    }
  }
  return res ;
}
